x-pack/plugins/profiling/public/components/flame_graphs_view/calculate_impact_estimates.test.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { calculateImpactEstimates } from './calculate_impact_estimates';
+
+describe('calculateImpactEstimates', () => {
+  it('calculates impact', () => {
+    expect(
+      calculateImpactEstimates({
+        countExclusive: 1000,
+        countInclusive: 1000,
+        totalSamples: 10000,
+        totalSeconds: 15 * 60, // 15m
+      })
+    ).toEqual({
+      annualizedCo2: 17.909333333333336,
+      annualizedCo2NoChildren: 17.909333333333336,
+      annualizedCoreSeconds: 1752000,
+      annualizedCoreSecondsNoChildren: 1752000,
+      annualizedDollarCost: 20.683333333333334,
+      annualizedDollarCostNoChildren: 20.683333333333334,
+      co2: 0.0005111111111111112,
+      co2NoChildren: 0.0005111111111111112,
+      coreSeconds: 50,
+      coreSecondsNoChildren: 50,
+      dollarCost: 0.0005902777777777778,
+      dollarCostNoChildren: 0.0005902777777777778,
+      percentage: 0.1,
+      percentageNoChildren: 0.1,
+    });
+  });
+});
```

Comment:
Looks good. Since this tests the values for a leaf node in a flamegraph, it would be useful to add a test case where `countExclusive` is less than `countInclusive` (likely a non-leaf node). In that case, any value `xyz` should be greater than `xyzNoChildren`.